src/components/Currency/index.module.scss

```diff
@@ -0,0 +1,4 @@
+.currencyComponent {
```

Comment:
Does currency component needs this styling by default?

Reply:
I put this just for demonstration, this is nothing but font-size/margins, could be edited any time. Should I remove it?